pkg/oc/cli/cmd/example.go

```diff
@@ -1,80 +0,0 @@
-package cmd
```

Comment:
any reason for removing this file altogether?

Reply:
> any reason for removing this file altogether?

it was dead.

Reply:
Oh, I thought we kept it as a template to guide new contributors?